### PR TITLE
CUJ Tweaks for I can manage a test version of my item.

### LINKED ISSUE #1471: [CWS CUJ] I can manage multiple versions of my item
[Friction Log](https://docs.google.com/document/d/119njHGcHGOIUc_p0AeFKvUlKWM6GiU1J37sf0DY8Pl4/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #1471 

CC: @CharyCh 

Changes proposed in this pull request:

New section to Payment and Distribution
- Add Instructions on how to manage a beta testing version and a production version in the CWS

**Staged links**
[Publishing a test version](https://deploy-preview-1490--developer-chrome-com.netlify.app/docs/webstore/cws-dashboard-distribution/#managing-a-test-version-and-a-production-version)

### CODE REVIEW

File: site/en/docs/webstore/cws-dashboard-distribution/index.md [hunk 1]
```diff
@@ -90,13 +90,35 @@ the Chrome Web Store. You can either:
 When you publish to the world, your item will be immediately visible to the region(s) you select,
 and it'll be visible in the store's search results.
 
+## Managing a test version and a production version
```
Comment (AmySteam) [inline site/en/docs/webstore/cws-dashboard-distribution/index.md:93]: @awfuchs Thoughts on this title? Suggestions are welcome 😁

Comment (awfuchs) [reply site/en/docs/webstore/cws-dashboard-distribution/index.md:93]: I think "Publishing a test version" works.

File: site/en/docs/webstore/cws-dashboard-distribution/index.md [hunk 2]
```diff
@@ -90,13 +90,35 @@ the Chrome Web Store. You can either:
...
+Before you submit a testing
+  version to the Chrome Web Store, follow these steps:
+
+  1.  Add the label "DEVELOPMENT BUILD" or “BETA” at the end of the name of your extension. 
+  2.  Declare the
+  purpose of your extension in the description: "THIS EXTENSION IS FOR BETA TESTING".
+
+{% Aside 'warning' %}
+
+Missing these steps can cause both your extensions to be taken down. For more information,
```
Comment (awfuchs) [inline site/en/docs/webstore/cws-dashboard-distribution/index.md:108]: Consider: "Make sure to add these labels. If you don't, then both extensions may be taken down."

**Review by awfuchs — APPROVED** [review]

File: site/en/docs/webstore/cws-dashboard-distribution/index.md (hunk 1 above)
Comment (dotproto) [inline site/en/docs/webstore/cws-dashboard-distribution/index.md:93]: This section looks like a good clarification on how to publish test versions. As discussed this morning, I think it's best for us to tackle white-labeling in a separate, more targeted CUJ.

Comment (AmySteam) [reply site/en/docs/webstore/cws-dashboard-distribution/index.md:93]: Great!

Comment (AmySteam) [reply site/en/docs/webstore/cws-dashboard-distribution/index.md:93]: Sounds good!

**Pull request merged**